crates/yttrium/src/chain_abstraction/client.rs

```diff
@@ -49,6 +71,213 @@ impl Client {
         }
     }
 
+    pub async fn get_route_ui_fields(
+        &self,
+        route_response: RouteResponseAvailable,
+        initial_transaction: Transaction,
+        currency: Currency,
+        // TODO use this to e.g. modify priority fee
+        _speed: String,
```

Comment:
What should client put for _speed?

Reply:
It's unused right now so you can put whatever, but it would presumably be an enum of fast, average, slow